Patches/Core/ThingDefs_Races/Races_Animal_BigCat.xml

```diff
@@ -60,14 +60,14 @@
 							</li>
 						</extraMeleeDamages>
 					</surpriseAttack>
-					<armorPenetrationSharp>0.09</armorPenetrationSharp>
+					<armorPenetrationSharp>0.16</armorPenetrationSharp>
 					<armorPenetrationBlunt>0.423</armorPenetrationBlunt>
 				</li>
 				<li Class="CombatExtended.ToolCE">
 					<capacities>
 						<li>Bite</li>
 					</capacities>
-					<power>14</power>
+					<power>18</power>
 					<cooldownTime>1.35</cooldownTime>
 					<linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
```

Comment:
Interestingly, the bite force in PSI can be found for a lot of predators, and size the units of blunt AP are in MPa, it's actually possible to get real stats for blunt AP. It could potentially be used with the sheets to sort of work backwards (to a limited extent) to find some of the other stats?

Reply:
I'll have to forgive me for not looking up the scientific data for animal bite forces, I generally don't, since I most patch fictional animals that have to be threathening to humans somehow.